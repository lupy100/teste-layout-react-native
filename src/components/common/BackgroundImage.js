// @flow
import styled from 'styled-components/native';

export default styled.Image`
  flex: 1;
  position: absolute;
  width: 100%;
  height: 100%;
`;
