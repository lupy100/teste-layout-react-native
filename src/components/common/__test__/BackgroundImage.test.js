import React from 'react';
import renderer from 'react-test-renderer';

import BackgroundImage from '../BackgroundImage';

test('renders correctly', () => {
  const tree = renderer
    .create(
      <BackgroundImage
        source={require('../../../../public/img/background.png')}
      />
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
