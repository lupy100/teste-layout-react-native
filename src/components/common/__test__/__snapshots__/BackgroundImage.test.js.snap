// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<Image
  source={
    Object {
      "testUri": "../../../public/img/background.png",
    }
  }
  style={
    Array [
      Object {
        "flexBasis": 0,
        "flexGrow": 1,
        "flexShrink": 1,
        "height": "100%",
        "position": "absolute",
        "width": "100%",
      },
      undefined,
    ]
  }
/>
`;
